lib/node_modules/@stdlib/math/base/special/minmax/manifest.json

```diff
@@ -0,0 +1,73 @@
+{
+    "options": {
+      "task": "build"
+    },
+    "fields": [
+      {
+        "field": "src",
+        "resolve": true,
+        "relative": true
+      },
+      {
+        "field": "include",
+        "resolve": true,
+        "relative": true
+      },
+      {
+        "field": "libraries",
+        "resolve": false,
+        "relative": false
+      },
+      {
+        "field": "libpath",
+        "resolve": true,
+        "relative": false
+      }
+    ],
+    "confs": [
+      {
+        "task": "build",
+        "src": [
+          "./src/main.c"
+        ],
+        "include": [
+          "./include"
+        ],
+        "libraries": [],
+        "libpath": [],
+        "dependencies": [
+          "@stdlib/math/base/napi/unary",
+          "@stdlib/math/base/special/minmax"
+        ]
```

Comment:
@AbhijitRaut04 You don't need to add the "@stdlib/math/base/special/minmax" in the `manifest.json` file. 

Dependencies in `manifest.json` are for external packages or modules that the code relies on. 

By listing `./include` in the include path, the `manifest.json` specifies that any header files within that directory i.e. `minmax.h` are available.

And since, the C implementation of `minmax` is not yet implemented listing it in the dependencies is causing the CI errors.

Reply:
@headlessNode  Removed that dependencies but still not working

Reply:
have you run `make install-node-addons NODE_ADDONS_PATTERN="math/base/special/minmax"` command on your system?

Reply:
![image](https://github.com/user-attachments/assets/12e21a5f-3de0-40e6-9070-e42169e21f16)
running this command gives error

Reply:
Maybe try compiling on a newer Node.js version and see if the issue persists. E.g., Node v20 or the latest LTS.